src/test/resources/features/personalInformation/edit_personalInformation.feature

```diff
@@ -0,0 +1,18 @@
+@PersonalInformation
+Feature: Edit Personal Information
+
+  Background:
+    Given I log in to Salesforce with a valid Account
+
+  @restorePersonalInformation
+  Scenario: Update personal information with the minimum values
+    When I go to settings
+    And I verify that the title of the panel is "Personal Information"
+    And I edit the form with the following data
```

Comment:
I think it could be: "I edit my personal information with the following data" to be more specific.